internal/driver/glfw/driver_windows.go

```diff
@@ -66,3 +67,19 @@ func setDisableScreenBlank(disable bool) {
 
 	syscall.Syscall(executionState.Addr(), 1, uintptr(uType), 0, 0)
 }
+
+const defaultDoubleTapDelay = 300 * time.Millisecond
```

Comment:
Could this const be left as applying to all drivers rather than adding it in multiple places - I think we should all fall back to the same value.

Reply:
Updated. Mobile is at 500 milliseconds by design I think since it's harder to tap quickly there, so I left it and defined a constant for each platform (desktop, web, mobile -_pre existing_) instead of a single Fyne-wide value